### PR TITLE
Added new command `slcli bandwidth pools-delete`

### PR DESCRIPTION
**Pull request by edsonarios**
Issue link: #1995 
```
softlayer-python/ (issue1995) $ slcli bandwidth pools-delete 1292861
Bandwidth pool with ID 1292861 was programmed to will delete
```

### CODE REVIEW

**Comment by ramkishor-ch** [comment]
Hi @edsonarios,

While testing the code changes in my local system,
I have tried to delete the Bandwidth pool with ID 1292861 but ID was not deleted.

Can you please check it.

Please find the screenshot for reference.

Screen Shot 1:
Screenshot 2023-06-22 at 12 58 16 PM

Screen Shot 2:
Screenshot 2023-06-22 at 12 58 27 PM

Thanks,
Ramkishor Chaladi.

**Comment by allmightyspiff** [comment]
Looks like this pool is being deleted, it just might take a while for that action to occur. We will want to add that information to the pools commands. I made https://github.com/softlayer/softlayer-python/issues/2002 to add this feature.

**Review by ramkishor-ch — APPROVED** [review]

File: SoftLayer/CLI/bandwidth/pools_delete.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+@click.argument('identifier')
+@environment.pass_env
+def cli(env, identifier):
+    """Delete bandwidth pool."""
+
+    manager = BandwidthManager(env.client)
+    bandwidth_pool = manager.delete_pool(identifier)
+
+    if bandwidth_pool:
+        env.fout(f"Bandwidth pool with ID {identifier} was programmed to will delete")
```
Comment (allmightyspiff) [inline SoftLayer/CLI/bandwidth/pools_delete.py:20]:
```suggestion
        env.fout(f"Bandwidth pool {identifier} has been scheduled for deletion.")
```

**Review by allmightyspiff — APPROVED** [review]
Looks good, just needed a minor text update and removed the `if` statement since it wasn't really needed.

File: SoftLayer/CLI/bandwidth/pools_delete.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+@click.command(cls=SLCommand)
+@click.argument('identifier')
+@environment.pass_env
+def cli(env, identifier):
+    """Delete bandwidth pool."""
+
+    manager = BandwidthManager(env.client)
+    bandwidth_pool = manager.delete_pool(identifier)
+
+        env.fout(f"Bandwidth pool {identifier} has been scheduled for deletion.")
```
Comment (allmightyspiff) [inline SoftLayer/CLI/bandwidth/pools_delete.py:19]:
```suggestion
    env.fout(f"Bandwidth pool {identifier} has been scheduled for deletion.")
```

**Review by allmightyspiff — APPROVED** [review]

File: tests/CLI/modules/bandwidth_tests.py [hunk 3]
```diff
@@ -61,6 +61,13 @@ def test_edit_bandwidth(self):
         self.assertEqual('MexRegionEdited', json_output['Name Pool'])
         self.assertEqual('MEX', json_output['Region'])
 
+    def test_delete_bandwidth(self):
+        result = self.run_command(['bandwidth', 'pools-delete', '123456'])
+        self.assert_no_fail(result)
+        self.assert_called_with('SoftLayer_Network_Bandwidth_Version1_Allotment', 'requestVdrCancellation')
+        json_output = json.loads(result.output)
+        self.assertEqual("Bandwidth pool with ID 123456 was programmed to will delete", json_output)
```
Comment (allmightyspiff) [inline tests/CLI/modules/bandwidth_tests.py:69]:
```suggestion
        self.assertEqual("Bandwidth pool 123456  has been scheduled for deletion.", json_output)
```

**Review by allmightyspiff — APPROVED** [review]

**Pull request merged**